sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/TriggerBindings/ServerlessHubOfT.cs

```diff
@@ -1,58 +1,64 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
-using System.IdentityModel.Tokens.Jwt;
-using System.Linq;
-using System.Security.Claims;
 using System.Threading.Tasks;
-using Azure.Core.Pipeline;
 using Microsoft.AspNetCore.SignalR;
 using Microsoft.Azure.SignalR.Management;
 
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
     [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1649:File name should match first type name", Justification = "https://github.com/Azure/azure-sdk-for-net/issues/17164")]
-    public abstract class ServerlessHub<T> where T : class
+    public abstract class ServerlessHub<T> : ServerlessHub where T : class
     {
-        private static readonly Lazy<JwtSecurityTokenHandler> JwtSecurityTokenHandler = new(() => new JwtSecurityTokenHandler());
         private readonly ServiceHubContext<T> _hubContext;
 
         /// <summary>
         /// Gets an object that can be used to invoke methods on the clients connected to this hub.
         /// </summary>
-        public IHubClients<T> Clients => _hubContext.Clients;
+        public new IHubClients<T> Clients => _hubContext.Clients;
 
         /// <summary>
         /// Get the group manager of this hub.
         /// </summary>
-        public GroupManager Groups => _hubContext.Groups;
+        public new GroupManager Groups => _hubContext.Groups;
 
         /// <summary>
         /// Get the user group manager of this hub.
         /// </summary>
-        public UserGroupManager UserGroups => _hubContext.UserGroups;
+        public new UserGroupManager UserGroups => _hubContext.UserGroups;
 
         /// <summary>
         /// Get the client manager of this hub.
         /// </summary>
-        public ClientManager ClientManager => _hubContext.ClientManager;
+        public override ClientManager ClientManager => _hubContext.ClientManager;
 
-        protected ServerlessHub(ServiceHubContext<T> serviceHubContext = null)
+        /// <summary>
+        /// Creates an instance of strongly typed serverless hub.
+        /// </summary>
+        /// <param name="serviceHubContext">Injects a service hub context.</param>
+        protected ServerlessHub(ServiceHubContext<T> serviceHubContext) : base(default(ServiceHubContext))
```

Comment:
This seems less than ideal, since it means that if someone has a `ServerlessHub<T>` typed as a `ServerlessHub`, the context will not be correct. What would happen in that case when using any of the properties that we are hiding - Clients, Groups, and UserGroups?

Reply:
Chatted offline, and the base type properties would actually throw nullref exceptions with the current implementation.